src/eva/vision/data/wsi/backends/openslide.py

```diff
@@ -58,3 +76,21 @@ def read_region(
             data[data[:, :, 3] == 0] = 255
 
         return data[:, :, :3]
+
+
+_conversion_factor_to_micrometer = {
+    "petameter": 10**21,
```

Comment:
could it be overkill here to go up to pentameter? probably in this context nothing above centimeter is usually used?

Reply:
maybe we can go up until meter (1e6), just to be sure?